styles/light/_light-theme.scss

```diff
@@ -445,6 +445,11 @@ aside .active > a:hover {
     white -2px -2px 3px, white -2px -2px 4px;
 }
 
+// Osano cookie consent overrides
+a.osano-cm-link {
```

Comment:
Our build doesn't compile the CSS on demand so you have to run `yarn run build-css` (or npm instead of yarn) from `styles/` and commit the changes to the repo when you make SCSS changes.

Reply:
Fixed